docs/rules/shebang.md

```diff
@@ -1,6 +1,6 @@
-# Require correct usage of shebang (`n/shebang`)
+# Require correct usage of hashbang (`n/shebang`)
 
-💼 This rule is enabled in the following [configs](https://github.com/eslint-community/eslint-plugin-n#-configs): ☑️ `flat/recommended`, 🟢 `flat/recommended-module`, ✅ `flat/recommended-script`, ☑️ `recommended`, 🟢 `recommended-module`, ✅ `recommended-script`.
+❌ This rule is deprecated. It was replaced by [`n/hashbang`](hashbang.md).
```

Comment:
We should probably date the removal of shebang for like 6 months from now, or the next major version?

Reply:
Not plan. It's fine to just keep the deprecated rules. let's just follow eslint's policy: https://eslint.org/docs/latest/use/rule-deprecation

Reply:
That makes sense! :+1: